Headlines/AppDelegate.swift

```diff
@@ -161,9 +161,10 @@ class AppDelegate: UIResponder,
     func application(_ application: UIApplication,
                      didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
 
+        #if !targetEnvironment(macCatalyst)
         //  https://www.herzbube.ch/blog/2016/08/how-hide-fabric-api-key-and-build-secret-open-source-project
         let resourceURL = Bundle.main.url(forResource: "fabric", withExtension: "apikey")
-
+        
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Canillitapp/headlines-iOS/blob/a5cc79d8b0bc8bc433e47d6a58e00624ade77840/Headlines/AppDelegate.swift#L167"></span>Lines should not have trailing whitespace.<br />
<code>trailing_whitespace</code> <code>AppDelegate.swift:167</code></td>
    </tr>
  </tbody>
</table>